test/src/test/java/org/corfudb/runtime/collections/CorfuStoreTest.java

```diff
@@ -0,0 +1,310 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.Message;
+
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.CorfuStoreMetadata;
+import org.corfudb.runtime.CorfuStoreMetadata.Timestamp;
+import org.corfudb.runtime.view.AbstractViewTest;
+import org.corfudb.test.SampleSchema.EventInfo;
+import org.corfudb.test.SampleSchema.Uuid;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.corfudb.test.SampleSchema.ManagedResources;
+
+/**
+ * Created by zlokhandwala on 2019-08-12.
+ */
+public class CorfuStoreTest extends AbstractViewTest {
+
+    @Test
```

Comment:
Can we add descriptions to all these tests.

Reply:
done.